version.go

```diff
@@ -1,4 +1,4 @@
 package captain
 
 // Version is the current version of this package. This needs to get bumped with every merge to 'main'.
-const Version = "v1.9.10"
+const Version = "v1.9.11"
```

Comment:
Is that idea that we merge releases/v1 into main after this so that the version is updated on main?

Reply:
We could. But I was thinking that `main` should be changed to a different or invalid version number instead so `captain --version` from main doesn't output a version that aligns with a particular release.

The simplest would probably be to have main on the "next" version with a `-dev` suffix (like `v1.9.12-dev`)

Reply:
IIRC abq uses `git describe` to tag in-between versions